scheduler/resource/host_manager.go

```diff
@@ -151,11 +152,22 @@ func (h *hostManager) RunGC() error {
 			return true
 		}
 
+		// If the host's elapsed exceeds twice the announcing interval,
+		// then leave peers in host.
+		elapsed := time.Since(host.UpdatedAt.Load())
+		if host.AnnounceInterval > 0 && elapsed-host.AnnounceInterval > host.AnnounceInterval {
```

Comment:
elapsed > host.AnnounceInterval

Reply:
@gaius-qi My idea is `elapsed > 2 * host.AnnounceInterval` then leave peers. To prevent overflow, I wrote `elapsed-host.AnnounceInterval > host.AnnounceInterval`.

Reply:
Changed to `elapsed > host.AnnounceInterval*2`, PTAL